compiler/src/iree/compiler/Pipelines/Options.h

```diff
@@ -89,6 +89,10 @@ struct HighLevelOptimizationOptions {
   // Strips debug assertions after any useful information has been extracted.
   bool stripAssertions = false;
 
+  // Specifies a set of passes to run immediately before dispatch formation
+  // intended for forming custom fusions.
+  std::string customFusionPassPipeline;
```

Comment:
Could you link to https://mlir.llvm.org/docs/PassManagement/#textual-pass-pipeline-specification in one of these locations? A sample string would also be appreciated, though the test this PR includes could suffice there.